soccer/src/soccer/.vscode/settings.json

```diff
@@ -0,0 +1,76 @@
+{
+    "files.associations": {
```

Comment:
Same comment as above. I would suggest that you add these changes to your global config of vscode. Be wary of what you push to the main branch.